apps/01PigLatin.js

```diff
@@ -4,12 +4,39 @@ var assert = require('assert');
 var prompt = require('prompt');
 prompt.start();
 
-
+function isVowel(letter) {
+  if(letter[0]==='a'|| letter[0]==='e'||letter[0]==='i'||letter[0]==='o'||letter[0]==='u')
+    return true;
+  }
 function pigLatin(word) {
+var vowelIndex = 0;
```

Comment:
Excellent! The only thing I want you to be mindful about Matt is indentation. It's very important!